test/test_torch.py

```diff
@@ -7590,6 +7602,17 @@ def test_rot90(self, device):
         self.assertEqual(torch.tensor([4, 3, 2, 1]).view(2, 2), data.rot90(2, [0, 1]))
         self.assertEqual(torch.tensor([3, 1, 4, 2]).view(2, 2), data.rot90(3, [0, 1]))
 
+        def test_complex_rot90(dtype):
+            data = torch.arange(1, 5, device=device).view(2, 2)
```

Comment:
Again, these tensors don't have the most interesting complex values. Comparing with NumPy will let you populate random values.